services/app/app/templates/portal/directory-sections.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="card mb-4">
+  <div class="card-body">
+    <p class="lead">Select your directory categories.</p>
+    <p>For each section below, select the directory categories your company should be featured in. You may select up to <code>{{ config.onlineDirectorySectionMax }}</code> categories per section.</p>
+    <p>{{ i18n "companyDetailsP3Label"}}</p>
+  </div>
+</div>
+
+<div class="row">
+  {{#each (get-edge-nodes this "model.websiteSites") as |site|}}
+    {{#if site.sections.edges.length}}
+      {{leadership/site site=site selectedIds=categories onUpdate=(action "update")}}
+    {{/if}}
+  {{/each}}
+</div>
+
+<div class="fixed-bottom fixed-right card">
+  <div class="card-body">
+    <button disabled={{submitDisabled}} class="btn btn-success pull-right" {{ action "showModal" }}>
+      {{entypo-icon "save"}} {{ i18n "saveChangesLabel"}}
+    </button>
+  </div>
+</div>
+{{submit-modal/categories
+  hash=hash
+  isOpen=isModalOpen
+  categories=categories
+  onComplete=(action "transitionToPortal")
+}}
```

Comment:
This needs to be created for directory sections, not reusing categories or leadership modals.